app/api/helpers/permissions.py

```diff
@@ -103,6 +103,27 @@ def decorated_function(*args, **kwargs):
     return decorated_function
 
 
+@second_order_decorator(jwt_required)
+def is_owner(f):
+    """
+    Allows only Owner to access the event resources.
+    :param f:
+    :return:
+    """
+
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        user = current_identity
+
+        if user.is_staff:
```

Comment:
Do we need this as this is only for an owner?

Reply:
Actually this is needed. It gives admin, superadmin access to as owner of the event. An admin and superadmin should have all the rights. That's why included this like other permissions